esmvaltool/_recipe.py

```diff
@@ -400,12 +400,8 @@ def _update_fx_settings(settings, variable, config_user):
                 fx_files_dict['sftgif'])
 
     for step in ('average_region', 'average_volume'):
-        if step in settings and settings[step].get('fx_files') is not False:
+        if step in settings and settings[step].get('fx_files') is not None:
```

Comment:
I would suggest something like this, because `var` is no longer needed now and the check `is not None` also does not add much, as `None` itself already evaluates to false.
```python
    for step in ('average_region', 'average_volume'):
        if settings.get(step, {}).get('fx_files'):
            settings[step]['fx_files'] = get_input_fx_filelist(
                variable=variable,
                rootpath=config_user['rootpath'],
                drs=config_user['drs'],
            )
```